test/Maybe/Maybe.js

```diff
@@ -65,6 +67,7 @@ var RightArb = function(arb) {
 var Compose = function(F, G) {
   var _Compose = function _Compose(x) {
     return {
+      constructor: _Compose,
```

Comment:
Agreed. I have several complaints about the current state of Fantasy Land, but I'm grateful to have a spec at all and I'm willing to implement it as faithfully as possible.